tests/ipython/testmagics.py

```diff
@@ -9,12 +9,15 @@
     raise SkipTest("Required dependencies not satisfied for testing magics")
 
 from holoviews.operation import Compositor
+from holoviews.plotting.comms import CommManager
```

Comment:
Lots of setting of ``comm_manager`` across the tests (made the same comment again below). Why is this needed?